MAINTAINERS.md

```diff
@@ -27,16 +29,25 @@ Review pull requests regularly, comment, suggest, reject, merge and close. Accep
 
 ### Triage Open Issues
 
-Review issues regularly and triage by labelling issues. For example, add "help wanted" to good issues for new community members. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
+Manage labels, review issues regularly, and triage by labelling them. For example, add "help wanted" to good issues for new community members and *blocker* for issues that scare you or need immediate attention. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
+
+### Be Responsive
+
+Respond to enhancement requests, and forum posts. Allocate time to reviewing and commenting on issues and conversations as they come in. 
 
 ### Maintain Overall Health of the Repo
 
-Keep the main branch at production quality at all times.
+Keep the `main` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches. 
```

Comment:
You've cut out the "how" here of how we want to do this.  I think it's important that we don't assume people know what the tags should be or what the branches should be.

So similar comment to above -- should the "hows" be in separate documents that live in github?  I had rolled the what you do and how you do it together in my maintainer doc.  But I'm not religious about keeping it that way, as long both are covered somewhere.

Reply:
We can't possibly spell the detailed how's in the template repository because YMMV across repos. How about accepting this as a start or suggest what additional text will bring it to par with minimum expectations?